pkg/collector/corechecks/servicediscovery/usm/tomcat.go

```diff
@@ -22,7 +23,8 @@ const (
 
 type (
 	tomcatExtractor struct {
-		ctx DetectionContext
+		ctx         DetectionContext
+		fileMatcher regexp.Regexp
```

Comment:
all the failed linter jobs are complaining about this unused field, so we will need to remove to get the CI green

Reply:
removed.